DSL/conditions.md

```diff
@@ -0,0 +1,288 @@
+# Declarative DSL to Check Conditions in JSON Schema
+
+This document provides an overview of a declarative Domain Specific Language (DSL) designed to check various conditions in a JSON schema. The DSL is structured as an array of conditions, with each condition containing three mandatory fields: `path`, `operation`, and `value`.
+
+## DSL Structure
+
+### Condition Fields
+
+- **path**: Specifies the path to the target in the schema where the operation will be performed.
+- **operation**: Specifies the type of operation to perform at the target path.
+- **value**: Specifies the value to be used in the operation.
+
+## Supported Operations
```

Comment:
```suggestion
## Operations
```

Just operations. Otherwise it implies there are "Unsupported Operations" too

Reply:
Also, I really think you should at one example for each operation you are presenting here

Reply:
Also, I think you should add a section right after this one purely about the pattern asterisk we support